static/src/javascripts-legacy/projects/common/modules/experiments/ab.js

```diff
@@ -243,9 +246,10 @@ define([
      */
     function registerCompleteEvent(complete) {
         return function initListener(test) {
-            var variantId = segmentUtil.variantIdFor(test);
 
-            if (segmentUtil.isInTest(test)) {
+            var variantId = getTestVariantId(test.id);
```

Comment:
Might be worth checking with someone who knows the a/b testing well that this is expected behaviour (although it seems right to me)